railties/test/configuration/dynamic_options_test.rb

```diff
@@ -7,15 +7,9 @@
 
 module RailtiesTest
   class DynamicOptionsTest < ActiveSupport::TestCase
-    class Configuration < Rails::Railtie::Configuration
```

Comment:
How about the one line change instead?

```suggestion
    class Configuration < Rails::Railtie::Configuration.dup
```

Reply:
I guess it works but inheriting from a `dup` is not necessarily something I'd like to leave behind as code that I wrote.
Also, replacing 8 lines with only 2 sounds more attractive to me.